automation/build-bin.ts

```diff
@@ -60,10 +61,10 @@ const standaloneZips: PathByPlatform = {
 	win32: dPath(`balena-cli-${version}-windows-${arch}-standalone.zip`),
 };
 
-const oclifInstallers: PathByPlatform = {
-	darwin: dPath('macos', `balena-${version}.pkg`),
-	win32: dPath('win32', `balena-${version}-${arch}.exe`),
-};
+const getOclifInstallersOriginalNames = (sha: string): PathByPlatform => ({
```

Comment:
Instead of patching oclif to make the naming the ones we want, we just get the name they create by default, this avoids a lot of mess up with the inner workings of oclif